### ISSUE TITLE
Implement the localized text with link helper function

### ISSUE THREAD

**Issue by rebmullin** [mozilla/addons-frontend#6983]
depends on:
~~#6881~~ 

This is a follow up to #6881.  We should also look into using the the helper function created in #6881 in the following places too:

https://github.com/mozilla/addons-frontend/blob/master/src/amo/components/AddonReviewManager/index.js#L85

https://github.com/mozilla/addons-frontend/blob/2018.11.15-1/src/amo/components/AddonReviewCard/index.js#L352-L368

verify - are there any other places this could be used?

note: this helper should be used for internal links only

**Comment by willdurand** [comment]
You can probably go the extra mile in #6881 given that it is where the helper function is introduced. That will help build a reusable helper function.

**Comment by rebmullin** [comment]
sorry @willdurand, I just saw your above comment.

**Comment by ioanarusiczki** [comment]
@rebmullin I verified the permalinks on AMO dev with FF63 (Win10) and review from an add-on detail page, view profile page and reviews page.  I did not see regressions. Also, review guidelines link works as expected.

![permalink](https://user-images.githubusercontent.com/33448286/49525334-3616cb80-f8b6-11e8-9ad0-eceeb29170eb.gif)

![permalink2](https://user-images.githubusercontent.com/33448286/49525346-3adb7f80-f8b6-11e8-8cd9-ae3c76cbd1d8.gif)
